src/main/kotlin/persistence/data/Lang.kt

```diff
@@ -0,0 +1,9 @@
+package persistence.data
+
+class Lang(
```

Comment:
this kind of thing should be a data class. I'd prefer it not be abbreviated. This class in general is also already being written by some other members of your team (perhaps try branching or merging with theirs)

Reply:
I talked to my team members about adding the json name annotations to the language data class in the common repo (like @Json(name = "lc")val slug: String) so that I could use that class instead, but they wanted to keep the moshi specific annotations in the jvm repo, so I instead create another language class and then map it to the data class labels from the common repo. What do you think?

Reply:
this class doesn't have any annotations in it

But anyway, yes, I suppose they're right. 
First, let's move all the retrofit stuff into an "api" package rather than persistence.
Next, have a subpackage called "model" for this kind of thing. You'll then need a mapper class to map between the common data classes and yours.